pkg/doc/verifiable/credential.go

```diff
@@ -800,3 +752,14 @@ func (vc *Credential) MarshalJSON() ([]byte, error) {
 
 	return byteCred, nil
 }
+
+// Presentation encloses credential into presentation.
+func (vc *Credential) Presentation() *Presentation {
+	vp := Presentation{
+		Context:    vc.Context,
+		Type:       []string{"VerifiablePresentation"},
```

Comment:
`VerifiablePresentation` better to have a constant for that

Reply:
Yeah, I will add a constant for that.